README.md

```diff
@@ -148,6 +194,32 @@ UnDeploy the controller from the cluster:
 make undeploy
 ```
 
+### Configuring integration tests
+The tests can be executed by running the following command:
+
+```sh
+make test
+```
+
+#### IAS stub
+Per default the integration tests use a stub for the IAS API.  
```

Comment:
👍🏼 it is really important that we document stubbing here.
What do you think that we force integration tests not to run with stubs? I think stub brings little value. Or did misunderstood anything?

Reply:
What should we add as documentation for the stubs? I'd not add a lot of details about the stubbing, since this is an implementation detail that may change.
I think using stubs for testing locally and on PRs is better, since the dependency to a 3rd party system might cause flakiness. When using real systems in the e2e tests that always something we should consider.
You can also just start by always using the real IAS app in the CI jobs and then depending on the reliability decide on additional steps. But for normal local development I think the stub should be the default, so IAS credentials are not a requirement during development.